dashboard/public/html/test_status.html

```diff
@@ -0,0 +1,39 @@
+<div>
+  <div layout="column" layout-align="start center" class="tst-status-div" layout-padding>
+    <md-card
+        class="file"
+        ng-repeat="file in vm.test.steps"
+        layout="row" layout-align="start center" layout-padding>
+      <md-card-content layout="column" layout-align="center start" flex>
+        <div class="file-status">
+          <div layout="row">
+            <div flex>
+            <h2 class="md-headline">{{ file.name }}</h2>
+            </div>
```

Comment:
@davidlimcheng Yep. I used it elsewhere. Not sure why I didn't use it here.
Have made the change.